[Problem 1](#Problem1) -- [Problem 2](#Problem2) -- [Problem 3](#Problem3) -- [Problem 4](#Problem4)

In [4]:
from scipy.stats import norm

In [9]:
norm.ppf(0.95)

1.6448536269514722

### Problem1

1. The dean from UCLA is concerned that the student's grade point averages have changed dramatically in recent years. The graduating seniors' mean GPA over the last five years is 2.75. The dean randomly samples 256 seniors from the last graduating class and finds that their mean GPA is 2.85, with a sample standard deviation of 0.65.

#### Explanation

mean gpa (upop) = 2.75  
sample size = 256  
mean gpa (usample) = 2.85  
sample Standard Deviation = 0.65  

* a  What would be the null and alternative hypothesis for this scenario?  

    **Null Hypothesis**: GPAs does not change dramatically, which implies, Mean GPA for Graduating seniors = 2.75  
    **Alternate Hypothesis** : GPAs changed dramatically  ,which implies Mean GPA for Graduation seniors != 2.75 
 

 * b) What would be the standard error for this particular scenario?

    **Standard Error**: = sample_Standard_Deviation/sqrt(sample size) = 0.65 /sqrt(256) = 0.65/16 =  0.04  
    
 
 * c) Describe in your own words how you would set the critical regions and what they would be at an alpha level of .05.  

     **Critical Region**: Since the null hypothesis is mean =2.75, the critial region lies on both the sides of the distribution, so it is a two-tailed test
     And for **alpha = 0.05**, the **confidence** would be **0.95** (, which is 95%), and the critiral region at **<=0.025** and **>0.975**  
 
 
 * d) Test the null hypothesis and explain your decision.  
    
    confidence interval : us +/ z*SE  
    z-upper_critical  = z at 97.5 = norm.ppf(0.975) = 1.96   
    z-lower_critical  = z at 0.025 = norm.ppd(0.025) = -1.96  
    z at usample = 2.85 = usample-upop/std.dev = (2.85 - 2.75)/0.04 = 0.1/0.04 = 10/4 = 2.5  
    z-observed falls under critical region
    
    **Answer**: Reject Null Hypothesis which implies, GPAs changed dramatically

### Problem2

2. The College bookstore tells prospective students that the average cost of its textbooks is Rs. 52 with a standard deviation of Rs.4.50. A group of smart statistics students thinks that the average cost is higher. To test the bookstore’s claim against their alternative, the students will select a random sample of size 100. Assume that the mean from their random sample is Rs. 52.80. Perform a hypothesis test at the 5% level of significance and state your decision.

#### Explanation

mean cost (upop) = Rs 52  
sample size = 100  
mean cost (usample) = 52.80  
population Standard Deviation = Rs 4.50  
significance = 5% = 0.05  
    **Null Hypothesis**: Average cost is not higher,which implies avg cost of books <=52  
    **Alternate Hypothesis** : Average cost is higher,which implies avg cost of books >52  
 
**Critical Region** lies on the right side of the distribution  
It is **positive-tailed test**  

 * b) What would be the standard error for this particular scenario?

    **Standard Error**: = population_Standard_Deviation/sqrt(sample size) = 4.50 /sqrt(100) = 4.5/10 =  0.45      
 
 * c) 
 Significance = 5%, and it is positive tailed test, so z-critical is at the 95%    
 
 * d) Test the null hypothesis and explain your decision.  
        
    z-critical for 0.95) = norm.ppf(0.95) = 1.64    
    z at usample =  usample-upop/std.error = (52.80-52)/0.45 = 0.80/0.45 = 1.77  
    z-observed falls under critical region  
    
**Answer**: Reject Null Hypothesis which implies, Avg cost of the price is greater than 52  

### Problem3

3. A certain chemical pollutant in the Genesee River has been constant for several years with a mean of 34ppm(parts per million) and standard deviation of 8ppm. A group of factory representatives whose companies discharge liquids into the river is now claiming that they have lowered the average with improved filtration devices. A group of environmentalists will test to see if this is true at the 1% level of significance. Assume that their sample of size 50 gives a mean of 32.5ppm. Perform a hypothesis test at the 1% level of significance and state your decision.
#### Explanation

mean pollution (upop) = 34 ppm  
population Standard Deviation = 8ppm  
sample size = 50  
sample mean pollution (usample) = 32.5ppm  
significance = 1% = 0.01 
* Frame Null and Alternate Hypothesis:
    **Null Hypothesis**: Mean pollution not lowered and remain >= 34ppm
    **Alternate Hypothesis** : Mean pollution has lowered and mean < 34ppm 
 
**Critical Region** lies on the left side of the distribution  
It is **negative-tailed test**  

 * b) What would be the standard error for this particular scenario?

    **Standard Error**: = population_Standard_Deviation/sqrt(sample size) = 8 /sqrt(50) = 1.13      
 
 * c) 
 Significance = 1%, and it is positive tailed test, so z-critical is at the 1%    
 
 * d) Test the null hypothesis and explain your decision.  
        
    z-critical for 0.01) = norm.ppf(0.01) = -2.32    
    z at usample =  usample-upop/std.error = (32.5-34)/1.13 = -1.37    
    z-observed falls under main region  
    
**Answer**: Fail to Reject Null Hypothesis which implies, The mean pollution has not lowered 

### Problem4

4. Carry out a one-tailed test to determine whether the population proportion of traveler’s check buyers who buy at least $2500 in checks when sweepstakes prizes are offered as at least 10% higher than the proportion of such buyers when no sweepstakes are on

Population 1 : With sweepstakes

N1 = 300

X1 = 120

S1 = 0.53

Population 2 : No sweepstakes

N2 = 700

X2 = 140

S2 = 0.20
#### Explanation
    
    
    
        
    
    
        
    
    
        
    
    
        
    
    
        
    
    
    
dd


### Problem5

A sample of 100 voters are asked which of four candidates they would vote for in an election. The number supporting each candidate is given below:-

    Higgins     Reardon     White     Charlton
        41        19         24          16

Do the data suggest that all candidates are equally popular? Chi-Square = 14.96, with 3 df < 0.05.

#### Explanation
